shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -1778,15 +1778,17 @@ private void sendAccessibilityEvent(@NonNull AccessibilityEvent event) {
   }
 
   /**
-   * Creates a {@link AccessibilityEvent#TYPE_WINDOW_STATE_CHANGED} and sends the event to Android's
-   * accessibility system.
+   * Informs TalkBack user about window name changes.
```

Comment:
```suggestion
   * Informs the TalkBack user about window name changes.
```

Apologies about the previous edit